templates/single.twig

```diff
@@ -4,27 +4,66 @@
 	<div class="content-wrapper">
 		<article class="post-type-{{ post.post_type }}" id="post-{{ post.ID }}">
 			<section class="article-content">
+				<div><span>{{ __('POST') }}</span>
+					{% for tag in post.tags %}
+						<span class="post-tag">
+							<a href="{{ tag.link }}">#{{ tag.title }}</a>
+						</span>
+					{% endfor %}
+				</div>
 				<h1 class="article-h1">{{ post.title|raw }}</h1>
 				<p class="blog-author">
-					<span>{{ __( 'By', domain ) }}</span><a href="{{ post.author.path }}"> {{ post.author.name }} </a><span>&bull;</span> {{ post.post_date|date }}
+					<span><i>{{ __( 'by', domain ) }}</i></span>
+					<a href="{{ post.author.path }}"><i>{{ post.author.name }} </i> </a><span><i>/</i>
+					</span> <i>{{ post.post_date|date }}</i>
 				</p>
 				<div class="article-body">
-					{{post.content|e('wp_kses_post')|raw}}
-				</div>
-			</section>
-		{% if ( 'closed' != post.comment_status ) %}
-			<section class="comments">
-				<div class="respond">
-					<h3 class="h2">{{ __( 'Comments', domain ) }}</h3>
-					{{ fn('comment_form') }}
+					{{ post.content|e('wp_kses_post')|raw }}
 				</div>
-				<div class="responses">
-					{% for cmt in post.get_comments() %}
-						{% include "comment.twig" with {comment:cmt} %}
-					{% endfor %}
+				<div class="article-author">
+					<div class="row">
+						<div class="col-md-6">
+							<div>
+								{{ __('ABOUT THE AUTHOR') }}
```

Comment:
Needs domain specified, why is it in all caps? I'd thought that kind of styling is done in CSS?